### PR TITLE
#1962: Added my name to credits page

### PR DESCRIPTION
**Pull request by XHoke04**
## Added my name to credits page

_Added my name to the credits page of finishline with a custom color._

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## Screenshots
Screenshot 2024-01-24 at 9 19 36 PM

_If you made UI changes you must post a screenshot of the whole page for each change in 1) a normal sized window and 2) the smallest possible window_

_If you did any manual testing (e.g., with Postman), put screenshots of the http request and before and after of the db_

_If none of this applies, you can delete this section_

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #1962)

### CODE REVIEW

File: src/frontend/src/components/ActionsMenu.tsx [hunk 1]
```diff
@@ -40,7 +40,19 @@ const ActionsMenu: React.FC<ActionsMenuProps> = ({ buttons, title = 'Actions' })
...
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'right'
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'right'
+        }}
+        onClose={handleDropdownClose}
+      >
```
Comment (Aaryan1203) [inline src/frontend/src/components/ActionsMenu.tsx:55]: You don't need to make any of these changes

File: src/frontend/src/pages/CreditsPage/CreditsPage.tsx [hunk 2]
```diff
@@ -208,7 +208,8 @@ const CreditsPage: React.FC = () => {
     { name: 'Visisht Kamalapuram', color: '#3083AA' },
     { name: 'Jay Jung', color: '##344739' },
     { name: 'Will Anderson', color: '#ebb145' },
-    { name: 'Chris Pyle', color: '#ADD8E6' }
+    { name: 'Chris Pyle', color: '#ADD8E6' },
+    { name: 'Xander Hoke', color: '#C50D93' }
```
Comment (Aaryan1203) [inline src/frontend/src/pages/CreditsPage/CreditsPage.tsx:212]: This looks good!

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
The only file you needed to change here was the credits page file. Make sure the other files are the same as they originally were.

Also, make sure to fill out the PR template with the changes that you made and delete the parts that you don't need

File: src/frontend/src/components/ActionsMenu.tsx (hunk 1 above)
Comment (XHoke04) [reply src/frontend/src/components/ActionsMenu.tsx:55]: Yeah, this was for another pr that was an accident lmao.

Comment (XHoke04) [reply src/frontend/src/components/ActionsMenu.tsx:55]: How do I remove the push from the other issue? In this case #955 from January 25th

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
It looks like you made changes to 49 files. It might be easier to checkout to develop, run git pull, and create a new branch to try it again

**Pull request closed without merging**